.env.example

```diff
@@ -19,12 +19,12 @@ LOG_STACK=single
 LOG_DEPRECATIONS_CHANNEL=null
 LOG_LEVEL=debug
 
-DB_CONNECTION=sqlite
-# DB_HOST=127.0.0.1
-# DB_PORT=3306
-# DB_DATABASE=laravel
-# DB_USERNAME=root
-# DB_PASSWORD=
+DB_CONNECTION=pgsql
+DB_HOST=db
```

Comment:
host, port, username and password are secrets data for the application. Don't put them in the repository.
You can take them from your docker-compose file on your local computer.
Keep them in docker-compose for now. But never explicitly specify them in the application configuration file.